api/lib/domain/models/Examiner.js

```diff
@@ -30,6 +36,25 @@ class Examiner {
       correctedAnswer.result = AnswerStatus.TIMEDOUT;
     }
 
+    if (Boolean(answer.isFocusedOut) !== hasAssessmentRecordedFocusedout) {
```

Comment:
**suggestion:** on peut utiliser le double point d'exclamation pour ça non ? 

```suggestion
    if (!!answer.isFocusedOut !== hasAssessmentRecordedFocusedout) {
```

Reply:
Sinon on fait ça
```suggestion
    if (answer.isFocusedOut ^ hasAssessmentRecordedFocusedout) {
```

Reply:
ou bien
```suggestion
    if (
      (answer.isFocusedOut && !hasAssessmentRecordedFocusedout) ||
      (!answer.isFocusedOut && hasAssessmentRecordedFocusedout)
    ) {
```

Reply:
Merci de voter

Reply:
:thinking: 
Qui connait un cas où il faut caster ainsi ?

Reply:
En bonus, on peut mettre `has` et `is` devant un booléen, avez-vous un critère pour décider ?

Reply:
Pas trop d'avis sur le `has`/`is`...

Reply:
Le !! fait le cast, donc revient exactement à ton `Boolean(_)`, juste une autre façon de l'écrire. Je voulais pas créer autant de remous